packages/features/ee/round-robin/roundRobinReassignment.ts

```diff
@@ -382,12 +383,13 @@ export const roundRobinReassignment = async ({
 
   // Handle changing workflows with organizer
   if (hasOrganizerChanged) {
-    const workflowReminders = await prisma.workflowReminder.findMany({
+    const scheduledWorkflowReminders = await prisma.workflowReminder.findMany({
       where: {
         bookingUid: booking.uid,
         method: WorkflowMethods.EMAIL,
+        scheduled: true,
+        OR: [{ cancelled: false }, { cancelled: null }],
```

Comment:
We only need to fix reminders for already scheduled emails. If email is not yet scheduled it will be scheduled in the crown job with the updated booking data